cam_status.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import argparse
+import sys
+import json
+from datetime import datetime, timedelta
+import time
+
+class test:
```

Comment:
I would change this to be more telling of what the class actually does.

Reply:
Update: Added comment detailing what the script does and made the class name more sensible.